conflagration/wrap.py

```diff
@@ -77,6 +79,9 @@ def multiparse(
         # Parse each file to a dictionary, and update the data dict with its
         # contents.
         for f in file_list:
+            if not os.path.exists(f):
```

Comment:
Would you mind moving this to the parse() function?  This function passes the file into the parse function, which is also used elsewhere, so that would be solving the same problem in two places at once.

Reply:
Or even better, ConfigFile.read()